spark-statsd-reporter/src/test/java/org/apache/spark/metrics/sink/statsd/MetricFormatterTest.java

```diff
@@ -0,0 +1,73 @@
+package org.apache.spark.metrics.sink.statsd;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.apache.spark.metrics.sink.statsd.MetricType.GAUGE;
+
+
+
+public class MetricFormatterTest {
+
+    @Test
+    public void simpleDriverMetricString() {
+        InstanceDetailsProvider instanceDetailsProviderMock = new InstanceDetailsProviderMock(
+                "834d77b5-a7b1-4c9d-9742-0f95d39d15e0-0002-driver-20190430095449-0001", "Test Spark App",
+                InstanceType.DRIVER, "aa31a823-2c7c-40e0-aef9-4b2a42adb461", "default");
```

Comment:
not relevant for the test correctness but for Driver instance type `instanceId` always equals to `applicationId`

Reply:
Good to know, but I'll leave it as is, if it's not relevant for correctness.